ec2/spark_ec2.py

```diff
@@ -1007,6 +1022,14 @@ def real_main():
         print >> stderr, "ebs-vol-num cannot be greater than 8"
         sys.exit(1)
 
+    # Limit naming to avoid breaking things, as we rely on the repo
```

Comment:
Can't we allow people to name the repo whatever they want? All we need to do is make sure the repo gets cloned to `spark-ec2` regardless of what it is actually called, no?